UPGRADING.md

```diff
@@ -210,85 +160,74 @@ Several existing methods have been renamed, a full list of which is shown below:
 
 See the [full documentation](https://docs.bugsnag.com/platforms/android) for more information.
 
-## BeforeNotify/BeforeSend/Callback
+## Event callbacks
```

Comment:
Users do not have prior knowledge about what an 'event callback' is. They will be familiar with `BeforeNotify/BeforeSend/Callback` and will search for those terms in the upgrade guide. I think we should therefore use the exact class name in at least part of each of the section titles to reduce confusion.